package.json

```diff
@@ -61,9 +58,8 @@
     "open": "0.0.5",
     "promzard": "0.3.0",
     "read": "1.0.7",
-    "request": "2.67.0",
```

Comment:
This is great but I'd like to make sure it doesn't happen again. Can we use [this rule](https://www.npmjs.com/package/eslint-plugin-dependencies) maybe?